### PR TITLE
Fix Settings for "PruebaWiki"

### PR DESCRIPTION
**Pull request by AlvaroMolinaCL**
Fix Settings for FlaggedRevs Extension on "PruebaWiki". Sorry for the repeated pull requests, but I've needed to see live configurations to know what they consist of and see how well they are. Thanks,

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -3591,6 +3586,15 @@
 			),
 		),
 		'+pruebawiki' => array(
+			'sysop' => array(
+				'confirmed',
```
Comment (Reception123) [inline LocalSettings.php]: These rights (confirmed, rollbacker and autopatrolled) are already in "default" so there is no need to re-add them.

File: LocalSettings.php [hunk 2]
```diff
@@ -4517,6 +4521,15 @@
...
```
Comment (Reception123) [inline LocalSettings.php]: Same comment as above.

File: LocalSettings.php (hunk 1 above)
Comment (AlvaroMolinaCL) [inline LocalSettings.php]: Done.

File: LocalSettings.php (hunk 2 above)
Comment (AlvaroMolinaCL) [inline LocalSettings.php]: Done.

**Pull request merged**